src/test/scala/com/yotpo/metorikku/test/metorikku-test-config-invalid-query.yaml

```diff
@@ -1,7 +1,9 @@
 metrics:
   - src/test/calculations/invalid_metric_query_test/
 inputs:
- employees: src/test/tables/employees.parquet
+ employees:
+  type: file
```

Comment:
Any way to make this the default behaviour? so you can still remain with:
`employees: src/test...`